slsim/Deflectors/DeflectorTypes/nfw_hernquist.py

```diff
@@ -0,0 +1,84 @@
+from slsim.Deflectors.DeflectorTypes.deflector_base import DeflectorBase
+from slsim.Deflectors.velocity_dispersion import vel_disp_composite_model
+from lenstronomy.Util import constants
+from lenstronomy.Cosmo.lens_cosmo import LensCosmo
+
+
+class NFWHernquist(DeflectorBase):
+    """Class of a NFW+Hernquist lens model with a Hernquist light mode.
+
+    required quantities in dictionary:
+    - 'halo_mass': halo mass in physical M_sol
+    - 'concentration': halo concentration
+    - 'e1_mass': eccentricity of NFW profile
+    - 'e2_mass': eccentricity of NFW profile
+    - 'stellar_mass': stellar mass in physical M_sol
+    - 'angular_size': half-light radius of stellar/light profile in radian
+    - 'e1_light': eccentricity of light
+    - 'e2_light': eccentricity of light
+    - 'z': redshift of deflector
+    """
+
+    def velocity_dispersion(self, cosmo=None):
+        """Velocity dispersion of deflector. Simplified assumptions on anisotropy and
+        averaged over the half-light radius.
+
+        :param cosmo: cosmology
+        :type cosmo: ~astropy.cosmology class
+        :return: velocity dispersion [km/s]
+        """
+        # convert radian to arc seconds
+        size_lens_arcsec = self.angular_size_light / constants.arcsec
+
+        m_halo, c_halo = self.halo_properties
+        # convert angular size to physical size
+        lens_cosmo = LensCosmo(z_lens=self.redshift, z_source=10, cosmo=cosmo)
```

Comment:
Do we need to use z_source here? if not, it might be useful to describe why we fix z_source = 10 here.

Reply:
ah, I only needed this class to get the angular diameter distance to the lens. I simplified it now using directly the astropy cosmology class to get to that